test/com/linkedin/drelephant/tuning/engine/SparkHBTParamRecommenderTestRunner.java

```diff
@@ -0,0 +1,74 @@
+package com.linkedin.drelephant.tuning.engine;
+
+import static common.DBTestUtil.initDBUtil;
+import static common.TestConstants.TEST_SPARK_HBT_PARAM_RECOMMENDER;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.util.HashMap;
+
+import models.AppResult;
+
+
+public class SparkHBTParamRecommenderTestRunner implements Runnable {
+
+  public void populateTestData() {
+    try {
+      initDBUtil(TEST_SPARK_HBT_PARAM_RECOMMENDER);
```

Comment:
Suggestion: While it's your wish, this unit test can be standalone and doesn't necessarily require DB interaction. You can create an AppResult object in code and pass it to SparkHBTRecommender and have the same effect.